whois-api/src/main/java/net/ripe/db/whois/api/rest/RestServiceHelper.java

```diff
@@ -125,6 +127,21 @@ public static WhoisResources createErrorEntity(final HttpServletRequest request,
         return whoisResources;
     }
 
+    public static String createErrorStringEntity(final HttpServletRequest request,
```

Comment:
Can you move this into the (Common) StreamingMarshal class? It's only used by the StreamingMarshal implementations. We should reduce the use of 'utility' code as much as possible.

Reply:
Done